playbook.md

```diff
@@ -1947,6 +1947,32 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+### Reporting our time
+
+Everyone who works on a client project at dxw digital is responsible for reporting the time they spend on projects. We usually work with our clients on a [time and materials basis](https://www.gov.uk/guidance/how-to-pay-for-digital-outcomes-and-specialists-services#time-and-materials), so it’s important we can accurately track and report how our time is spent on client projects.
+
+We use [10,000ft](https://app.10000ft.com/) to schedule our work, which helps us to forecast our capacity over future weeks and months in a reliable way. 
+
+Our Commercial Operations team update the schedule to reflect any planned leave, which helps us to look ahead - this is recorded as “Out of office". Each member of our delivery team uses the tool to record the days they work on client projects on a weekly basis.
+
+We track our time at the same level of granularity that we bill it – for almost everyone in the team, that’s half days or whole days.
+
+#### What to do when reporting your time
+
+When reporting our time, we follow these guidelines:
+
+* If you worked on a named project (client or internal), record your time as that.
+* If you worked on something else, leave the time blank or zero out what you were scheduled to work on.
+* If you were off sick, record your time as "Sick".
+* If you had unplanned leave, record your time as "Out of office".
```

Comment:
I think it's helpful to also include planned leave here, since you're also confirming what was scheduled, and want to be leave scheduled "Out of office" as that.